modules/examples/inner-destinations/tls-test-validation/tls-test-validation.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020 One Identity
+ * Copyright (c) 2020 Laszlo Budai <laszlo.budai@outlook.com>
```

Comment:
You can use your own name in the newly introduces files.
Please also update the year.